ocw_import/management/commands/import_ocw_course_sites.py

```diff
@@ -1,9 +1,11 @@
 """ Import OCW course sites and content via ocw2hugo output """
+import json
 import pydoc
 
 from django.conf import settings
-from django.core.management import BaseCommand
+from django.core.management import BaseCommand, CommandError
 from mitol.common.utils.datetime import now_in_utc
+from six.moves import input
```

Comment:
**Question:** Do we need six? We're not trying to support Python 2...

Reply:
1.  I was confused about how `input` works in management commands, and it seems you don't need to use the `six` compatibility layer to get that to work.  I switched it to just use the native `input` function.

2.  The server side filtering is definitely a massive time sink and we should bake filtering into the actual use of the S3 API, but that's out of scope for this PR.  I made issues for that and another thing I noticed that we should change now that editing is turned off in Plone here:

https://github.com/mitodl/ocw-studio/issues/1217
https://github.com/mitodl/ocw-studio/issues/1218